clarkio/ttv-chat-light
# New sound effect: Its A Bold Strategy Cotton, Lets See If It Pays Off For Em

## Purpose

- A new funny sound effect from dodgeball

## Does this introduce a breaking change?


## Pull Request Type

What kind of change does this Pull Request introduce?



- [x] Other... Please describe: Sound effect

## How to Test



- Get the code

```
git clone git@github.com:clarkio/ttv-chat-light.git
cd ttv-chat-client
git checkout [branch-name]
npm install
```
run the !bold command and verify the sound effect is played

### What to Check

Verify that the following are valid

- ...

## Other Information
